ci

```diff
@@ -61,11 +61,21 @@ popd
 
 # Once we have setup the environment, made errors fatal, and self-upgraded (all
 # of which should never fail), we try to report any unexpected failure.
+# Scrub < > and & out of message as Slack documentation says those are only 3 characters that need to be encoded as HTML entities
+# only doing the "message" part as things like author or channel won't have those
 #  $1: channel
 #  $2: message
 post_to_slack () {
-  curl -X POST --data-urlencode "payload={\"channel\": \"$1\", \
-    \"username\": \"dzomo\", \"text\": \"$2\", \
+
+SLACK_MESSAGE=$2
+
+temp1=$(echo $SLACK_MESSAGE | sed -e :1 -e "s@\(Message.*\)\&\(.*Author\)@\1xxxxx\2@;t1")
+temp2=$(echo $temp1| sed -e :1 -e "s@\(Message.*\)xxxxx\(.*Author\)@\1\&amp;\2@;t1") # work around for a known sed bug where second & causes problems
+temp3=$(echo $temp2 | sed -e :1 -e 's@\(Message.*\)<\(.*Author\)@\1\&lt;\2@;t1') 
+SCRUBBED_SLACK_MESSAGE=$(echo $temp3 | sed -e :1 -e 's@\(Message.*\)>\(.*Author\)@\1\&gt;\2@;t1')
```

Comment:
ok... can you comment on why this isn't just:
```
local slack_message="$2"
slack_message="${slack_message/&/&amp;}"
slack_message="${slack_message/</&lt;}"
slack_message="${slack_message/>/&gt;}"
slack_message="${slack_message/\"/&quot;}"
```
?

Nits:
- only variables intended to be global are in uppercase, and it's good to use the `local` keyword as much as possible
- indentation within a function